### ISSUE TITLE
Sprite.set_texture(0) ignores scale

### ISSUE THREAD

**Issue by DavidLP** [pvcraven/arcade#318]
## Bug Report
When switching the texture of a sprite the scale attribute is not used and the scale is [deduced from the texture scale attribute](https://github.com/pvcraven/arcade/blob/master/arcade/sprite.py#L476). But the [texture scale attribute is never set to the actual scale](https://github.com/pvcraven/arcade/blob/master/arcade/sprite.py#L117) for the original texture.
### Actual behavior:
Changing back to the original texture: ```sprite.set_texture(0) ``` changes the scale to ```1```.
### Expected behavior:
The scale of the original texture should be used.
### Steps to reproduce/example code:
``` python
sprite = arcade.Sprite(filename, scale = 0.1)   # now scale = 0.1
sprite.append_texture(TEXTURE_OBJECT)
sprite.set_texture(1)
sprite.set_texture(0)  # now scale = 1
```
## Enhancement request:
To fix this one could just add this line
``` python
self._texture.scale = scale
```
in the code [here](https://github.com/pvcraven/arcade/blob/master/arcade/sprite.py#L119).

**Comment by pvcraven** [comment]
Try the new beta: 2.0.0b5 and let me know if it is fixed.

**Comment by pvcraven** [comment]
Closed with 2.0 release.
